iocore/net/P_SSLNetVConnection.h

```diff
@@ -260,6 +260,17 @@ class SSLNetVConnection : public UnixNetVConnection
   /// Set by asynchronous hooks to request a specific operation.
   SslVConnOp hookOpRequested;
 
+  bool
+  getClientVerifyEnable()
+  {
+    return clientVerifyEnable;
+  }
+  void
+  setClientVerifyEnable(int enable)
```

Comment:
IF you are truly changing this configuration away from being a 0|1 (which is an incompatible change technically, and therefore possibly should wait until 8.0) you can keep this as int. But then I would insist that we don't name it "enabled", and the getter function  also must return int.